modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -45,5 +45,7 @@ endif::[]
 --
 1. Egress firewall is also known as egress network policy in OpenShift SDN. This is not the same as network policy egress.
 
-2. Does not support egress rules and some `ipBlock` rules.
+2. Network policy for OpenShift SDN does not support egress rules and some `ipBlock` rules.
+
+3. Egress router for OVN-Kubernetes supports _redirect mode_ only.
```

Comment:
```suggestion
3. Egress router for OVN-Kubernetes supports only *redirect mode*.
```
First use of a term can be styled with italics. Emphasis is bold. See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
@kalexand-rh I've always used only italics for emphasis, not bold. I usually see bold reserved for GUI items. Is there a reason that we deviate from the IBM style guide on this?

Reply:
This is the first use of the term "redirect mode" on this page.  Shouldn't it be italics?

Reply:
> @kalexand-rh I've always used only italics for emphasis, not bold. I usually see bold reserved for GUI items. Is there a reason that we deviate from the IBM style guide on this?

I don't see the context for why we chose this deviation in the Git history, so your guess is as good as mine.

> This is the first use of the term "redirect mode" on this page. Shouldn't it be italics?

I've only seen italics on first use is almost always for terms that you're explaining. You're emphasizing the term, not explaining it.

Reply:
@kalexand-rh, as usual, you are correct and thank you.